rfcs/xxxx-rfc-process/text.md

```diff
@@ -0,0 +1,60 @@
+# RFC Process <img style="float: right;" src="./imgs/monkey_mascot.jpg" width="200">
+
+**Authors:** [Arathi Mani](mailto:arathi.mani@chanzuckerberg.com)
+
+**Approvers:** [Timmy Huang](mailto:thuang@chanzuckerberg.com), [Marcus Kinsella](mkinsella@chanzuckerberg.com), [Brian Raymor](mailto:braymor@chanzuckerberg.com)
```

Comment:
I do not believe that I should be an approver of a *Dev* software practice. I'm happy to be a reviewer and advise based on my experience with the use of github in standards organizations and my related research on PIP and Rust when developing the DCP RFC Process.

Based on the template, I'm not anticipating that PM or UX would be creating RFCs for product plans or UX designs?

Reply:
Thank you; I added you as an approver because of your experience, but have moved you to Reviewer now :) 

No, I have not assumed here that PM or UX would be creating RFCs, but happy to have a conversation if that is ideally the case. I'd love for either PM or UX to spearhead that conversation and/or create templates.